linux_os/guide/system/auditing/package_audit_installed/osbuild_blueprint/shared.toml

```diff
@@ -0,0 +1,5 @@
+# platform = multi_platform_all
+
+[[packages]]
```

Comment:
Out of curiosity, especially for a `package_*_installed` checks, why not add this to the template? Does enabling templates require additional work outside of the scope of this PR? Just curious :)

Reply:
Hi, yesa, this should be definitely templated in the future and it requires a bit of additional work. I would do it as a different PR.